packages/@okta/i18n/src/properties/login.properties

```diff
@@ -811,7 +811,10 @@ oie.okta_verify.signed_nonce.label = Use Okta Verify on this device
 oie.okta_verify.enroll.force.upgrade.title = Update Okta Verify
 oie.authenticator.app.non_fips_compliant_enrollment_device_incompatible = The device used to set up Okta Verify does not meet your organization’s security requirements because it is not FIPS compliant. Contact your administrator for help.
 oie.authenticator.app.non_fips_compliant_enrollment_app_update_required = The Okta Verify version on the device used does not meet your organization’s security requirements. To add your account, update Okta Verify to the latest version, then try again.
-
+oie.authenticator.app.method.push.verify.enable.biometrics = We received your response, but your organization requires biometrics. To proceed, ensure that your device supports biometrics, then add your account and enable biometrics when prompted.
```

Comment:
Do users have to add the account again ? I guess users get to this point after push is sent and biometrics is not sent back to backend ? So technically account is already added just that we need to enable biometrics.

Reply:
Yeah, I dont think they need to add account again

Reply:
@nbories-okta should we rephrase the text accordingly ?

Reply:
Will update as per new text